ast/misc.go

```diff
@@ -2074,6 +2075,19 @@ func (n *TableOptimizerHint) Restore(ctx *RestoreCtx) error {
 		ctx.WriteKeyWord(n.QueryType.String())
 	case "memory_quota":
 		ctx.WritePlainf("%d M", n.MemoryQuota)
+	case "read_consistent_storage":
+		ctx.WritePlain(n.StoreType.O)
```

Comment:
```suggestion
		ctx.WritePlain(n.StoreType.String())
```

Reply:
If we use `String()`, it may causes `TIFLASH` ---> <code>\`tiflash\`</code>. Then the SQL hint will like <code>READ_CONSISTENT_STORAGE(\`tiflash\`[\`t1\`, \`t2\`])</code>, that's strange.

Reply:
Only `ctx.WriteName()` will add those backquotes. BTW you should use `ctx.WriteKeyWord()` for case-insensitive keyword-like output.

Reply:
OKay. Thanks for your advisement.

Reply:
??? I'm confused.
``` golang
func (cis CIStr) String() string {
	return cis.O
}
```

Reply:
@foreyes My fault. I thought you suggested using the `WriteName`.